### ISSUE TITLE
False positives

### ISSUE THREAD

**Issue by ivorpad** [envato/envato-theme-check#13]
1. I got a warning and the ID is not referenced anywhere in my stylesheets.

```
WARNING: The theme is using `#wpadminbar`. Hiding the admin bar is not allowed.
```

2. `REQUIRED: The <title> tags can only contain a call to wp_title(). Use the wp_title filter to modify the output`

I'm not adding anything extra to `<title>` tags so, this requirement shouldn't be displayed.

```
function theme_prefix_display_the_title() {
  ?>
    <title><?php wp_title( '|', true, 'right' ); ?></title>
  <?php
}
add_action( 'wp_head', 'theme_prefix_display_the_title' );
```

3. I've received also a lot of warnings like these and `phpcs` is in another directory and not part of the theme:

```
WARNING: fclose was found in the file wpcs/vendor/squizlabs/php_codesniffer/CodeSniffer/Reports/Gitblame.php File operations should use the WP_Filesystem methods instead of direct PHP filesystem calls.
Line 95: fclose($handle);
WARNING: fclose was found in the file wpcs/vendor/squizlabs/php_codesniffer/CodeSniffer/Reporting.php File operations should use the WP_Filesystem methods instead of direct PHP filesystem calls.
Line 245: fclose($this->_tmpFiles[$report]);
WARNING: fclose was found in the file wpcs/vendor/squizlabs/php_codesniffer/CodeSniffer/Fixer.php File operations should use the WP_Filesystem methods instead of direct PHP filesystem calls.
Line 274: fclose($fixedFile);
```

**Comment by scottparry** [comment]
@ivorpad Can you provide me with the said theme so I can check it out?

**Comment by ivorpad** [comment]
Hey @scottparry,

Fortunately it was something I had in my theme's folder. My bad. 

Closing this one.
